src/main/java/core/basesyntax/service/impl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.time.LocalDateTime;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    private final String path;
```

Comment:
I'd also recommend passing path as an argument to the method instead of creating a separate path field